ci-operator/step-registry/storage/conf/wait-for-csi-migration/storage-conf-wait-for-csi-migration-commands.sh

```diff
@@ -94,5 +94,59 @@ function wait_for_nodes() {
     done
 }
 
+function nodes_stable() {
+    # Check that the node is Ready
+    oc wait --for=condition=Ready=True node --all --timeout=0 || return 1
+    # Check the node is schedulable
+    if oc get node -o yaml | grep "unschedulable"; then
+        return 1
+    fi
+}
+
+function cluster_stable() {
+    oc wait --all --for=condition=Progressing=False clusteroperators.config.openshift.io --timeout=0 || return 1
+    oc wait --all --for=condition=Available=true clusteroperators.config.openshift.io --timeout=0 || return 1
+    oc wait --all --for=condition=Degraded=false clusteroperators.config.openshift.io --timeout=0 || return 1
+}
+
+function wait_for_stable_cluster() {
+    # A cluster is considered stable when:
+    # - all nodes are Ready
+    # - all nodes are schedulable
+    # - CVO is Available=true, Progressing=false, Degraded=false
+    # - all the checks above are stable for 1 minute
+    local COUNT=1
+    local STABLE_COUNT=1
+    while true; do
+        if nodes_stable ; then
```

Comment:
If one of the nodes is `unschedulable` or not `Ready`, this  function (`nodes_stable`) returns `1`, otherwise it doesn't return anything. In this case, isn't this if/else swapped?

Reply:
Not sure if I miss anything, IMO zero exit code means OK / true in shell.

Reply:
Ah, OK then